ui/components/DashboardComponent.js

```diff
@@ -660,265 +646,294 @@ class DashboardComponent extends React.Component {
-              <Select
-                value={self.state.activeMeshScanNamespace[mesh.name]}
-                onChange={(e) =>
-                  self.setState((state) => ({ activeMeshScanNamespace : { ...state.activeMeshScanNamespace, [mesh.name] : e.target.value }, }))
-                }
-              >
-                {self.state.meshScanNamespaces[mesh.name] &&
-                  self.state.meshScanNamespaces[mesh.name].map((ns) => <MenuItem key={ns.uniqueID} value={ns}>{ns}</MenuItem>)}
-              </Select>
-            )}
-          </>
-        )
-      },
-    }
-
-    if (Array.isArray(components) && components.length)
-      return (
-        <Paper elevation={1} style={{ padding : "2rem", marginTop : "1rem" }}>
-          <MuiThemeProvider theme={theme.palette.type == "dark" ? configurationTableThemeDark() : configurationTableTheme() }>
-
-            <MUIDataTable
-              className={this.props.classes.datatable}
-              title={
-                <>
-                  <div style={{ display : "flex", alignItems : "center", marginBottom : "1rem" }}>
-                    <img src={mesh.icon} className={this.props.classes.icon} style={{ marginRight : "0.75rem" }} />
-                    <Typography variant="h6">{mesh.tag}</Typography>
-                  </div>
-                </>
-              }
-              data={components}
-              options={options}
-              columns={columns}
-            />
-          </MuiThemeProvider >
-        </Paper>
-      );
-
-    return null;
-  };
-
+  // const MeshCard = (mesh, components = []) => {
+  //   let componentSort = "asc";
+  //   let versionSort = "asc";
+  //   let proxySort = "asc";
+  //   let tempComp = [];
+  //   const { theme } = props;
+
+  //   components
+  //     .filter(comp => comp.namespace === props.activeMeshScanNamespace[mesh.name])
+  //     .map(component => tempComp.push(component));
+
+  //   components = tempComp;
+
+  //   const switchSortOrder = type => {
+  //     if (type === "componentSort") {
+  //       componentSort = componentSort === "asc" ? "desc" : "asc";
+  //       versionSort = "asc";
+  //       proxySort = "asc";
+  //     } else if (type === "versionSort") {
+  //       versionSort = versionSort === "asc" ? "desc" : "asc";
+  //       componentSort = "asc";
+  //       proxySort = "asc";
+  //     } else if (type === "proxySort") {
+  //       proxySort = proxySort === "asc" ? "desc" : "asc";
+  //       componentSort = "asc";
+  //       versionSort = "asc";
+  //     }
+  //   };
+
+  //   const columns = [
+  //     {
+  //       name : "name",
+  //       label : "Component",
+  //       options : {
+  //         filter : false,
+  //         sort : true,
+  //         searchable : true,
+  //         setCellProps : () => ({ style : { textAlign : "center" } }),
+  //         customHeadRender : ({ index, ...column }, sortColumn) => {
+  //           return (
```

Comment:
why so much commented out code, are these have no use?

Reply:
if not then give description why they are commented

Reply:
these functions weren't used anywhere in the component. That's why I commented them

Reply:
@sudhanshutech  these were functions that were there before I changed to functional component. I thought these functions would be used later with a newer version. So, I just commented these out and removed the pieces of code that were converted